gnmvidispine/vs_item.py

```diff
@@ -812,10 +812,12 @@ def streaming_import_to_shape(self, filename, transferPriority=500, throttle=Tru
         url = "/item/{0}/shape/raw"
         if rename is None: rename=os.path.basename(filename)
         
-        self.chunked_upload_request(io.FileIO(filename),os.path.getsize(filename),chunk_size=1024*1024,
+        request = self.chunked_upload_request(io.FileIO(filename),os.path.getsize(filename),chunk_size=1024*1024,
```

Comment:
don't bother setting `request` here, just return the value of `self.chunked_upload_request`